### PR TITLE
Reorganize Hello World Page

### PR DESCRIPTION
**Pull request by alexkrolick**
Emphasizes the text that explains the organization of the Main Concepts section and the introductory code by moving the codeblock to the very top and adding a "following along" heading for the install steps

### CODE REVIEW

File: content/docs/hello-world.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ redirect_from:
   - "docs/getting-started-zh-CN.html"
 ---
 
-The easiest way to get started with React is to use [this Hello World example code on CodePen](codepen://hello-world). You don't need to install anything; you can open it in another tab and follow along as we go through examples. If you'd rather use a local development environment, check out the [Installation](/docs/try-react.html) section.
+Getting Started With React
```
Comment (gaearon) [inline content/docs/hello-world.md]: Do we ever need this heading?

Comment (alexkrolick) [reply content/docs/hello-world.md]: Looks a bit weird if the next like becomes the subheading

Comment (gaearon) [reply content/docs/hello-world.md]: I don't think I agree.

It's not a subheading, it's a leading paragraph. We use it this way on all the other pages. I don't think we use this style for a subheading.

Comment (alexkrolick) [reply content/docs/hello-world.md]: Ok

**Pull request merged**